packages/x-components/src/components/column-picker/base-column-picker-list.vue

```diff
@@ -47,7 +56,10 @@
    * @public
    */
   @Component({
-    components: { BaseEventButton }
+    components: { BaseEventButton },
+    model: {
+      event: 'change'
```

Comment:
This is to match the other similar components (like the sort selectors, or the dropdown)